release-notes/release-notes53.md

```diff
@@ -29,6 +29,45 @@ The following new features and enhancements were added for the {{site.productnam
 
 The following premium plugins updates were released alongside {{site.productname}} 5.3.
 
+### Enhanced Media Embed
+
+The {{site.productname}} 5.3 release includes the following accompanying changes affecting **Self-hosted Essential, Professional, Premium, and Enterprise customers only**:
```

Comment:
Should this say `Enterprise` here? As we really only have the other 3 now and the last was called "Pro" in the download page.

Reply:
fixed by https://github.com/tinymce/tinymce-docs/pull/1482/commits/a371f8e8bf84dbb8784ef55e828d1af6ddb5b69f